### PR TITLE
Created Google Login, removed facebook and changed readme

### PR DESCRIPTION
**Pull request by calebji123**
## Changes

Added Google login button and functionality.
Removed facebook login.
Database should still work fine with old facebook profiles.

## Checklist

- [x] Pull request has descriptive context useful to reviewer. Screenshots are attached as necessary.
- [x] Pull request title matches Linear ticket.
- [x] Ticket has been claimed on Linear
- [ ] Pull Request has been assigned to yourself
- [ ] Pull request has been approved by CodeOwner
- [ ] Pull request has been approved by one other student

### CODE REVIEW

**Review by marianmwang — APPROVED** [review]
looks good.

File: client/src/routes/layout.js [hunk 1]
```diff
@@ -6,16 +6,19 @@ import { CCF_THEME } from "../actions/theme";
...
 const Layout = () => {
   return (
-    <ThemeProvider theme={CCF_THEME}>
-      <LinkSection />
-      <Box padding="1rem" margin="4rem 0.5rem 1rem 0.5rem">
-        <WelcomeHeader />
-        <Outlet />
-      </Box>
-    </ThemeProvider>
+    <GoogleOAuthProvider clientId="858566898033-jau549hlkcrroj0fhgg0upj53anhbcde.apps.googleusercontent.com">
```
Comment (marianmwang) [inline client/src/routes/layout.js]: is it fine to just have this in code? should we put it in .env..? idk

Comment (calebji123) [reply client/src/routes/layout.js]: just did! good catch i forgot abt that

**Comment by Jnwong116** [comment]
@calebji123 looks like you just have to fix the merge conflict in `README`. 

Although you might want to see if you can move the Google secret into the actual `.env` file. From what I remember, there's something wrong with `dotenv` or something but maybe you'll have better luck. Otherwise yea go ahead and merge.

**Pull request closed without merging**